docker-compose.yml

```
@@ -1,41 +1,37 @@
+version: "3.9"
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)